src/rpc/core_rpc_server.cpp

```diff
@@ -616,6 +616,23 @@ namespace cryptonote
       return true;
     }
 
+    unsigned int concurrency_count = std::thread::hardware_concurrency() * 4;
```

Comment:
I realized after posting that I was taking the "the user is stoopid let me fix it" mentality to programming. I think `* 4` is a reasonable tradeoff. Adding a comment about it will help prevent someone from "fixing" it in the future.

@ashleypt